force-app/main/default/classes/DMLOperations.cls

```diff
@@ -81,10 +87,11 @@ public with sharing class DMLOperations {
     public static void updateContactLastName(Id contactId, String newLastName) {
         // Retrieve the Contact from the database
         // [SELECT Id FROM Contact WHERE Id = :contactId]
-
+        Contact existingContact = [SELECT Id FROM Contact WHERE Id = :contactId];
         // Update the Contact's last name
-
+        existingContact.LastName = 'Smith';
         // Update the Contact in the database
+        update existingContact;
```

Comment:
Good, in  production code you might want to think about how you handle a situation where that query does not return anything.